src/integrationTest/java/uk/gov/hmcts/darts/audio/service/AudioTransformationServicehandleKedaInvocationForMediaRequestsGivenBuilder.java

```diff
@@ -28,7 +28,7 @@
 @Setter
 @Scope(scopeName = SCOPE_PROTOTYPE)
 @SuppressWarnings("MethodName")
-public class AudioTransformationServiceProcessAudioRequestGivenBuilder {
+public class AudioTransformationServicehandleKedaInvocationForMediaRequestsGivenBuilder {
```

Comment:
I guess these 2 files in the change list with `Servicehandle` should be renamed to be `ServiceHandle`...?